src/pages/index.js

```diff
@@ -42,12 +26,25 @@ export default function Home({ data }) {
   );
 }
 
-export async function getStaticProps() {
+// export async function getStaticProps() {
+//   const res = await fetch(
+//     `https://www.thecocktaildb.com/api/json/v1/1/filter.php?c=Cocktail`,
+//   );
+//   const data = await res.json();
+//   return {
+//     props: {
+//       data: data.drinks,
+//     },
+//     revalidate: 10,
+//   };
+// }
+
+// Server Side
+export async function getServerSideProps() {
   const res = await fetch(
```

Comment:
let's abstract this http calls to a different folder/file, to make it independent of the implementation and easy to test.